changelog/unreleased/enhancement-add-admin-space-details-sidebar

```diff
@@ -0,0 +1,6 @@
+Enhancement: Add space details for admins
```

Comment:
You can just add your PR link to the existing changelog item for the spaces management. It makes more sense from a user perspective since we're releasing it together as one feature.